lib/src/values/logic_value.dart

```diff
@@ -640,8 +645,11 @@ abstract class LogicValue implements Comparable<LogicValue> {
   /// matter what the length. When leading, 'Z' indicates one or more 'z'
   /// bits to fill the first radix character.
   /// - `9'bz zzzz zzz = 9'hZZZ`
-  ///
-  String toRadixString({int radix = 2, int chunkSize = 4}) {
+  String toRadixString(
+      {int radix = 2, int chunkSize = 4, String sepChar = '_'}) {
+    if (radixStringChars.contains(sepChar)) {
+      throw Exception('separation character invalid');
```

Comment:
We're trying to (at least in ROHD) move to more specific types of exception (see `exceptions` directory) than a generic `Exception`.  Can you see if one meets your needs or add a new one?